src/content/docs/blog/tauri-board-elections-2024.md

```diff
@@ -0,0 +1,49 @@
+3. **Apply prior to July 5th, 2024** by emailing [board@tauri.app](mailto:board@tauri.app), messaging the `@board` role on the [Tauri Discord](https://discord.com/invite/tauri), or by directly messaging Jacob Bolda on Discord (`@jacobbolda`) or email ([jacob@tauri.app](mailto:jacob@tauri.app)).
+
+Voting will take place starting on July 7th through July 12th, 2024 where Tauri Working Group Members will cast their votes. We're expecting to announce the results on or before July 16, 2024. If elected, we'll ask you to sign [this pledge](https://dracc.commonsconservancy.org/0016/) if you haven't done so previously.
+
+If you have any questions or would like more information please reach out by emailing [board@tauri.app](mailto:board@tauri.app), messaging the `@board` role on the [Tauri Discord](https://discord.com/invite/tauri), or by directly messaging Jacob Bolda on Discord (`@jacobbolda`) or email ([jacob@tauri.app](mailto:jacob@tauri.app)).
+
+## Interested In Getting Involved?
+
+If joining the Board isn't for you, we are always looking to enable involvement at any level within Tauri. The documentation for v2 is a large focus of the remaining effort. See the issues within the [tauri-apps/tauri-docs repo](https://github.com/tauri-apps/tauri-docs). Additionally reach out and follow along within Discord. Add a `@notify-*` role to stay apprised of news and questions from the team, or follow along and ask questions in the `#dev` or `#docs` channels under the `CONTRIBUTORS` section.
+
+Prepared for some more responsibility? A Domain in the Working Group represents a specific area of interest.
+
+- **Development**: Developing and maintaining the Tauri software
+- **Community**: Looks after the broader community resources and the public presence of Tauri
+- **Governance & Guidance**: Involved with all things organizational in nature
+- **Operations**: Responsible for all the tooling and infrastructure Tauri needs to get work done
+
+This gives clear entry points for a new member of the Working Group to jump into right away. It also allows us to continue growing our core competency of core Tauri development while giving additional focus to all areas which support the Tauri organization as a whole. Each of these domains will be represented by Domain Members and Domain Leads, but membership is not mutually exclusive.
+
+All domains within Tauri are equally important to providing the best experience possible, as well as the wonderful support of the community. ❤️
+
+---
+
+We hope that these changes provide the foundation to continue the journey the Tauri Working Group has been on while also catalyzing the continued growth and sustainability of the organization. You can learn more about these updates in the [Governance and Guidance repo on GitHub](https://github.com/tauri-apps/governance-and-guidance).
```

Comment:
Nit: refers to "these changes" but besides a possible change of Board Directors the model remains otherwise the same.